src/pages/Success.svelte

```diff
@@ -18,96 +19,110 @@
 	let allFunded = false;
 	let cashlinks: Array<CashlinkStore> = [];
 
+	const updateStore = () => {
+		console.log("Updating Store");
+		cashlinks.forEach((x) => {
+			const index = $cashlinkArray.findIndex((cashlink) => cashlink.tx.transactionHash === x.tx.transactionHash);
```

Comment:
again this function?? maybe move it to a lib/shared file?

Reply:
not worth it yet, but good idea if used again